tests/integration_tests.py

```diff
@@ -51,13 +48,7 @@ def token(self):
 AUTH_CLIENT = client_auth(auth_header())
 
 
-def upload_cubes(data):
-    fname = tempfile.mktemp("segy")
-    segyio.tools.from_array(fname, data)
-
-    with open(fname, "rb") as f:
-        meta = scan.scan(f)
-
+def upload_cube(fname, meta):
```

Comment:
Why is the scanning step moved out of this function?

Reply:
An attempt to make the code less ugly. The fixture needs to return the guid of the cube. The scanning does not belong in the upload and it was hard to come up with good names; meaning the functions did too much. Appreciate a better solution.

Reply:
A simpler but uglier attempt pushed